python/amici/sbml_import.py

```diff
@@ -1611,7 +1617,9 @@ def printWithException(self,math):
 
         """
         try:
-            return self.codeprinter.doprint(math)
+            ret = self.codeprinter.doprint(math)
+            ret = re.sub(r'(^|\W)M_PI(\W|$)', r'\1amici::pi\2', ret)
```

Comment:
why are there both M_PI and amici::pi again?

Reply:
I did not find a way to convince CCodeprinter to directly use amici::pi. So here M_PI is replaced by amici::pi.